it/test/uk/gov/hmrc/testuser/repository/TestUserRepositorySpec.scala

```diff
@@ -152,7 +154,7 @@ class TestUserRepositorySpec extends AsyncHmrcSpec with BeforeAndAfterEach with
       val timeLA1 = getLastAccessFor(query)
       val timeCO1 = getCreatedOnFor(query)
 
-      Thread.sleep(100)
```

Comment:
does it want the old test to check the times are == when the time difference is less than the period? Or is it risking making slow jenkisn cause test failures

Reply:
Good spot.  Added another test for this.